File: src/TestCenter.Models/Assessment.cs
using System;
using System.Collections.Generic;

namespace TestCenter.Models
{
    public class Assessment
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public List<string> Labels { get; set; }
        public List<Question> Questions { get; set; }

        public Assessment(Guid id, string name = "", string description = "")
        {
            this.Id = id;
            this.Name = name;
            this.Description = description;
        }
    }
}

File: app/TestCenter.Ionic/src/models/assessment.ts
import { Question } from './question';

export class Assessment {
  public id: string;
  public name: string;
  public shortName: string;
  public description: string;
  public categories: Array<string>;
  public questions: Array<Question>;

  constructor(name: string, shortName: string,
    description: string, categories: Array<string>) {
    this.name = name;
    this.shortName = shortName;
    this.description = description;
    this.categories = categories;
  }
}
